Jenkinsfile

```diff
@@ -660,13 +660,13 @@ def withBeatsEnv(boolean archive, Closure body) {
   ]) {
     deleteDir()
     unstash 'source'
-    if(os == 'linux'){
+    if(isDockerInstalled()){
       dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
       // FIXME workaround until we fix the packer cache
       // Retry to avoid DDoS detection from the server
       retry(3) {
         sleep randomNumber(min: 2, max: 5)
-        sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
+        //sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
```

Comment:
Is this intentional? This PR seems like it's just about adding the credentials and not about disabling docker pulls.

Reply:
this PR removes the need of the workaround, so I've removed those lines